examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -742,6 +742,15 @@ void WindowManager::ButtonEventHandler(uint8_t button, uint8_t btnAction)
     }
 
     AppTask::GetAppTask().PostEvent(&event);
+
+// Since there is no release interrupt for button1 posting release event after press event
+#ifdef SIWX_917
```

Comment:
Needs to be removed, this is part of another PR

Reply:
remove , will add in a separate PR.